lib/ncr_dispatcher.rb

```diff
@@ -28,6 +28,7 @@ def on_proposal_update(proposal)
     }
 
     proposal.observers.each{|observer|
+      next if modifier and observer.id == modifier.id  # https://www.pivotaltracker.com/story/show/100957216
```

Comment:
Mind adding a comment to the effect of "don't notify the person who triggered the notification"?